javascript/react/intro-react/one-way-data-binding.md

```diff
@@ -27,19 +27,17 @@ aspects:
 ---
 ## Content
 
-**React** encourages **unidirectional data flow** by design, making it the canonical example of this approach nowadays.
+**React** encourages **unidirectional data flow** by design, making it the canonical example of this approach nowadays. Along with two-way data binding, these approaches describe the way your application keeps the `view` in sync with the data `model`.
 
-Along with two-way data binding, these approaches describe the way your application keeps the `view` in sync with the data `model`.
-
-In two-way data binding the `model` is dynamically bound to the `view` such that one can change the other and vice-versa.
+In two-way data binding, the `model` is dynamically bound to the `view` such that one can change the other and vice-versa.
```

Comment:
Since React only has one-way data binding, maybe add two-way data binding info as a footnote?

Reply:
Changed paragraphs to:

**React** encourages **unidirectional data flow** by design and lets you describe the way your application keeps the `view` in sync with the data `model` using an HTML-like language called JSX. This process is called **one-way data binding**[1].

In **one-way data binding**, the `model` is the *single source of truth*. Instead of changing the `model` directly when a **UI field** changes, the `model` is notified by the intended change and updates the *state* itself. Afterwards, the `view` is updated to match the change.

[1: Two-way data binding]
Although React uses one-way data binding, there is also two-way data binding. By this we mean that the `model` is dynamically bound to the `view` such that one can change the other and vice-versa:
- when you update a property in the model, the UI is also updated
- when you update the UI, the data model is also changed